AWS-Infrastructure-Setup.md

```diff
@@ -0,0 +1,75 @@
+# AWS Infrastructure Setup
+
+In order for the included demos, integration tests, and even your own Sumerian Hosts apps to be runnable you will need to set up a few things in your AWS account. The steps below will guide you through creating a **Cognito Identity Pool** that allows the demo applications to talk to two AWS services—**Amazon Polly** and **Amazon Lex**. You'll also create an **Amazon Lex chatbot** that powers one of the demos.
```

Comment:
Is the full name "Amazon Sumerian Hosts" and the short name "Sumerian Hosts" or is the full name "Sumerian Hosts" and the short name "Hosts?" There are some usages of all three of these, so want to double-check.

Reply:
We decided to use "Hosts" as the short name. I will update the docs.